packages/grafana-prometheus/package.json

```diff
@@ -0,0 +1,151 @@
+{
+  "author": "Grafana Labs",
+  "license": "Apache-2.0",
+  "name": "@grafana/prometheus",
+  "version": "10.4.0-pre",
+  "description": "Grafana Prometheus Library",
+  "keywords": [
+    "typescript"
+  ],
+  "sideEffects": false,
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/grafana/grafana.git",
+    "directory": "packages/grafana-prometheus"
+  },
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "publishConfig": {
+    "main": "dist/index.js",
+    "module": "dist/esm/index.js",
+    "types": "dist/index.d.ts",
+    "access": "public"
+  },
+  "files": [
+    "dist",
+    "./README.md",
+    "./CHANGELOG.md",
+    "LICENSE_APACHE2"
+  ],
+  "scripts": {
+    "build": "tsc -p ./tsconfig.build.json && rollup -c rollup.config.ts",
+    "clean": "rimraf ./dist ./compiled ./package.tgz",
+    "typecheck": "tsc --emitDeclarationOnly false --noEmit",
+    "prepack": "cp package.json package.json.bak && node ../../scripts/prepare-packagejson.js",
+    "postpack": "mv package.json.bak package.json"
+  },
+  "dependencies": {
+    "@emotion/css": "11.11.2",
+    "@floating-ui/react": "0.26.8",
+    "@grafana/data": "workspace:*",
+    "@grafana/experimental": "1.7.8",
+    "@grafana/faro-web-sdk": "^1.3.6",
+    "@grafana/runtime": "workspace:*",
```

Comment:
Shouldn't this also be a peerDependency so it resolves to the singleton in Grafana core?

Reply:
This makes sense to me.